pkg/parquet/reader.go

```diff
@@ -156,6 +158,7 @@ func (r *s3Reader) ReadDateAsync(ctx context.Context, datetime time.Time, target
 				stop = true
 				return
 			}
+			S3FilesToDelete = append(S3FilesToDelete, file)
```

Comment:
this should be rather something like `readS3Files` and then depending on the options delete the fiiles on read date right?

Reply:
That is a very good point. Will update the name